code/client/src/App.js

```diff
@@ -93,20 +102,46 @@ class App extends Component {
           </Switch>
 
           {backdrop}
-          <Route path="/about" component={About} />
-          <Route path="/account" component={Account} />
-          <Route path="/blogs" component={Blogs} />
-          <Route path="/contact" component={Contact} />
-          <Route path="/disclaimer" component={Disclaimer} />
-          <Route path="/faq" component={Faq} />
-          <Route path="/register" component={Register} />
-          <Route path="/search_locations" component={SearchLocations} />
-          <Route path="/login" component={Login} />
-          <Route path="/donate" component={Donate} />
+          <Switch>
+            <Route path="/about" component={About} />
+            <Route path="/account" component={Account} />
+            <Route path="/blogs" component={Blogs} />
+            <Route path="/contact" component={Contact} />
+            <Route path="/disclaimer" component={Disclaimer} />
+            <Route path="/faq" component={Faq} />
+            <Route path="/register" component={Register} />
+            <Route path="/search_locations" component={SearchLocations} />
+            <Route path="/login" component={Login} />
+            <Route path="/donate" component={Donate} />
+            <Route path="/forgot" component={Reset} />
+            <Route path="/:token" component={Reset_token} />
+          </Switch>
+
+
+
         </div>
       </Router>
     );
   }
 }
 
+//<Switch> <Route path="/:token" children={<Reset_token />} /></Switch>
```

Comment:
Is this chunk of code still being used? If we don't need it anymore we can just remove it.

Reply:
There is a block of commented code below this. Might be more clear viewing these in the files changed tab instead of here

Reply:
Yes, I'll get rid of that.

Reply:
I've just removed that line and the other lines below that.